src/mkdocstrings/docstrings.py

```diff
@@ -211,7 +211,18 @@ def read_parameters_section(self, lines, start_index):
         parameters = []
         block, i = self.read_block_items(lines, start_index)
         for param_line in block:
-            name, description = param_line.lstrip(" ").split(":", 1)
+            try:
+                name_with_type, description = param_line.lstrip(" ").split(":", 1)
+                paren_index = name_with_type.find("(")
+                if paren_index != -1:
+                    # name (type)
+                    name = name_with_type[0:paren_index].strip()
```

Comment:
I like that, it could help implementing #27

Reply:
Definitely, would be easy to add a regex to extract the type if it were present.